CHANGELOG.md

```diff
@@ -3,7 +3,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [2.0.4] - TBD
+## [3.0.0] - TBD
+### Added
+- Support for running metastore in EC2 nodes.
```

Comment:
`Support for running metastore in EC2 nodes` --> `Support for running metastore running Hive Metastore on EC2 nodes.`